src/stim/simulators/error_analyzer.test.cc

```diff
@@ -2420,9 +2420,11 @@ TEST(ErrorAnalyzer, too_many_symptoms) {
     ASSERT_EQ("", check_catch<std::invalid_argument>("max supported number of symptoms", [&] {
                   ErrorAnalyzer::circuit_to_detector_error_model(symptoms_20, true, false, false, 0.0, false, true);
               }));
-    ASSERT_EQ("", check_catch<std::invalid_argument>("max supported number of symptoms", [&] {
-                  ErrorAnalyzer::circuit_to_detector_error_model(symptoms_20, false, false, false, 0.0, false, true);
-              }));
+    ASSERT_EQ(
```

Comment:
Copy paste this case. It should pass when not decomposing (as you've done here) and fail if decomposing (which just requires setting one of the bool arguments to true).

Reply:
There was already a case to check for a failure when `decompose_errors = True`, so I just had to modify the test to accept the expected behavior when `decompose_errors = False`. Is that what you had in mind?

Reply:
Ah, okay, then that's fine as is.